snuba/state/cache/redis/scripts/set.lua

```diff
@@ -11,22 +20,22 @@
 -- responsible for executing. If it doesn't exist or does not match the current
 -- task ID, we must have overrun the timeout.
 -- TODO: This may still be able to safely set the cache value?
-local task_id = redis.call('GET', KEYS[3])
-if not task_id or task_id ~= ARGV[1] then
+local task_id = redis.call('GET', task_id_key)
+if not task_id or task_id ~= task_id then
     return {err="invalid task id"}
 end
 
 -- Update the cache value.
-if ARGV[3] ~= nil then
-    redis.call('SETEX', KEYS[1], ARGV[4], ARGV[3])
+if value ~= nil then
+    redis.call('SETEX', value_key, value_ttl, value)
 end
 
 -- Move the data from the waiting queue to the notify queue.
-redis.call('RENAME', KEYS[2], KEYS[4])
-redis.call('EXPIRE', KEYS[4], ARGV[2])
+redis.call('RENAME', wait_queue_key, notify_queue_key)
+redis.call('PEXPIRE', notify_queue_key, notify_queue_ttl)
```

Comment:
Do we still need `PEXPIRE` here now that the TTL for exceptions is 1 second?

Reply:
another great catch. Weird that the tests didn't catch this. I will add a test before re-submitting for review